gx_tests_unemployment_extracted_data_expects.py

```diff
@@ -0,0 +1,147 @@
+import great_expectations as gx
+
+from great_expectations.checkpoint import Checkpoint
+
+context = gx.get_context()
+
+my_connection_string = "sqlite:///./db.sqlite"
+datasource_name = "my_datasource"
+
+datasource = context.sources.add_sqlite(
+    name=datasource_name, connection_string=my_connection_string
+)
+
+datasource.add_table_asset(
+    name="unemployment", table_name="unemployment"
+)
+
+batch_request = datasource.get_asset("unemployment").build_batch_request()
+
+expectation_suite_name = "unemployment_validation"
+context.add_or_update_expectation_suite(expectation_suite_name=expectation_suite_name)
+validator = context.get_validator(
+    batch_request=batch_request,
+    expectation_suite_name=expectation_suite_name,
+)
+
+print(validator.head())
+
+# Completeness
+validator.expect_column_values_to_not_be_null(column="FIPStxt")
+validator.expect_column_values_to_not_be_null(column="State")
+validator.expect_column_values_to_not_be_null(column="Area_name")
+validator.expect_column_values_to_not_be_null(column="Year")
+validator.expect_table_columns_to_match_set([
+    "FIPStxt",
+    "State",
+    "Area_name",
+    "Year",
+    "unemployment_rate",
+], exact_match=True
+)
+
+# Consistency
+validator.expect_column_values_to_be_of_type(column="FIPStxt", type_="INTEGER")
+validator.expect_column_values_to_be_of_type(column="State", type_="TEXT")
+validator.expect_column_values_to_be_of_type(column="Area_name", type_="TEXT")
+validator.expect_column_values_to_be_of_type(column="Year", type_="INTEGER")
+validator.expect_column_values_to_be_of_type(column="unemployment_rate", type_="REAL")
+
+# Uniqueness
+validator.expect_column_unique_value_count_to_be_between(column="FIPStxt", min_value=0, max_value=72153)
```

Comment:
I think this expectation does not makes too much sense, try to use another column or another example of uniqueness